modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/change/tracking/spi/display/JournalArticleCTDisplayRenderer.java

```diff
@@ -123,6 +117,13 @@ public String getVersionName(JournalArticle journalArticle) {
 	public String renderPreview(DisplayContext<JournalArticle> displayContext)
 		throws Exception {
 
+		String preview = renderDisplayPagePreview(
```

Comment:
@pei-jung 
This method will not work for journal article because in the base class the AssetEntry and retrieved by the model's primaryKey. But for journal article we need to use the resourcePrimKey